addons/voice/org.openhab.voice.marytts/src/main/java/org/openhab/voice/marytts/internal/MaryTTSAudioStream.java

```diff
@@ -89,7 +89,7 @@ private InputStream getWavHeaderInputStream(int length) throws IOException {
         byte format = 0x10; // PCM
         byte bits = 16;
         byte channel = 1;
-        int srate = 48000;
+        long srate = (this.audioFormat != null) ? this.audioFormat.getFrequency() : 16000l;
```

Comment:
> Only the US uses 48kHz, so it is 1:2 for 16kHz

Haha, we are not bargaining here ;-)
It is still "US/english" the default for openHAB, no matter of how many other voices there are :-)

> Anyway us voice is working well, because for real text synthetics, the audioFormat is set correctly

Which also means "german voice is working well", if we set the default to 48k, doesn't it?